content/docs/manual/endpoints.md

```diff
@@ -31,6 +31,11 @@ Endpoint authentication is a scenario where the endpoint owner needs to specify
 
 ![Endpoint authentication](/docs-assets/endpoint-auth.png)
 
+### Endpoint Owner ID
+Endpoint Owner ID is a unique ID for identifying a group of endpoints. It is useful for fanning out an event to multiple endpoints and creating portal link for multiple endpoints.
```

Comment:
We should add that "it is typically set by you the user" and it should typically have some sort of meaning in their internal systems providing logical grouping of endpoints on their end.